jukebox.go

```diff
@@ -62,6 +64,22 @@ func (j *Jukebox) start() {
 					fmt.Printf("Description: %s\n", playlistItem.description)
 				}
 
+				var timeoutduration time.Duration
+				//from json file
+				if playlistItem.playtime > 0 {
+					timeoutduration = time.Duration(playlistItem.playtime) * time.Second
+				}
+				//from command prompt

```

Comment:
I jigged this around - the Jukebox has a default playtime, and this is now set from a constant, so it's only the playlistItem.playtime that needs to be tested as a special case